README.markdown

```diff
@@ -46,7 +46,14 @@ or Wget:
 wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.31.3/install.sh | bash
 ```
 
-<sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc`, `~/.profile`, or `~/.bashrc`).</sub>
+<sub>The script clones the nvm repository to `~/.nvm` and adds the following source lines to your profile (`~/.bash_profile`, `~/.zshrc`, `~/.profile`, or `~/.bashrc`).</sub>
+
+```
+export NVM_DIR="/home/user/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"  # This loads nvm
+```
+
+Where 'user' in the above NVM_DIR exported path is substituted for the current logged in user.
```

Comment:
`/home/user` is not a standard path for `$HOME` - if instead you use `$HOME`, it should work everywhere.